libs/deer/rust-toolchain.toml

```diff
@@ -1,3 +1,3 @@
 [toolchain]
 channel = "nightly-2023-03-20"
-components = ['rust-src', 'clippy', 'rustfmt', 'miri']
+components = ['rustfmt', 'clippy', 'miri']
```

Comment:
note (to me): will require rust-src in the near future for integration tests of the derive macros.